### PR TITLE
docs: added note for the TAG option

### PR DESCRIPTION
**Pull request by yuriw**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
commit and PR titles should be `doc/dev: added ...`

**Comment by ljflores** [comment]
Rendered doc: https://ceph--51146.org.readthedocs.build/en/51146/dev/release-process/

**Comment by ljflores** [comment]
Anyone have an idea why the bullet point I added is bold? Not sure how to fix it (see https://ceph--51146.org.readthedocs.build/en/51146/dev/release-process/)

@anthonyeleven

**Comment by anthonyeleven** [comment]
That's ... curious.

My guess is that the terminal colon on line 47 is causing the preceding text to be interpreted as a heading or some other non-body item.  I might suggest removing that colon and see if the rendering changes.

@zdover23 might be able to say more authoritatively, he's better clued into some of the quirks of our toolchain.

**Comment by ljflores** [comment]
Thx @zdover23 and @anthonyeleven! I'll push a fix.

**Comment by zdover23** [comment]
> Ok, now it's not bold anymore. Thanks!

@ljflores: Qapla.

**Comment by dmick** [comment]
@zdover23 My Klingon is not what it used to be.  Not that it ever used to be anything, really.

File: doc/dev/release-process.rst [hunk 1]
```diff
@@ -41,11 +41,15 @@ Hotfix Release Process Deviation
...
-2. ``git cherry-pick -x`` the necessary hotfix commits (Note: only "cherry-pick" must be used)
-3. ``git push -f origin quincy-release``
-4. Notify the "Build Lead" to start the build.
-5. The "Build Lead" should set ``RELEASE_TYPE=HOTFIX`` instead of ``STABLE``.
+1. Check out the most recent tag. For example, if we're releasing a hotfix on top of 17.2.3, ``git checkout -f -B quincy-release tags/v17.2.3``.
+2. ``git cherry-pick -x`` the necessary hotfix commits (Note: only "cherry-pick" must be used).
+3. ``git push -f origin quincy-release``.
+4. Verify the commits in the ``$release-release`` branch:
+
+   1. To check against the previous point release (if we are making 17.2.4, this would be 17.2.3), run ``git log --pretty=oneline --no-merges tags/v17.2.3..origin/quincy-release``. Verify that the commits produced are exactly what we want in the next point release.
```
Comment (rzarzynski) [inline doc/dev/release-process.rst:49]: :+1:

**Pull request merged**